src/stories/DfxVideoHelpModalContent.tsx

```diff
@@ -0,0 +1,26 @@
+import StyledVideoGrid, { VideoSourcesProps } from './StyledVideoGrid';
+
+export interface DfxVideoHelpModalContentProps {
+  videoSources: VideoSourcesProps[];
+  numCols?: number;
+  title?: string;
+  description?: string;
```

Comment:
Are these really not required?

Reply:
yes, you can totally display a content grid and want just the title or just the description or neither one ...